docker/peft-gpu/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN source activate peft && \
     git+https://github.com/huggingface/accelerate \
     peft[test]@git+https://github.com/huggingface/peft
 
-RUN python3 -m pip install --no-cache-dir bitsandbytes
+RUN python3 -m pip install --no-cache-dir bitsandbytes optimum auto-gptq
```

Comment:
Maybe we need to install optimum from main?

Reply:
Or maybe wait until the release on optimum side ? @fxmarty said that it is not advised to install from source.